docs/contributing/building.rst

```diff
@@ -135,6 +135,63 @@ or source it into your shell,
 
    . $BUILD_DIR/python_env.sh
 
+Resolving Common Build Issues
+=============================
```

Comment:
I think this is useful, however it does duplicate some elements from other parts of the documentation. Can you add links to more details on each of these issues? For instance in 1 you could link to the git page, and in 3 you could link to the conda setup and build instructions.